examples/cron/compound-staking-cron/src/lib.rs

```diff
@@ -53,6 +62,18 @@ pub struct BankAmount {
     amount: String,
 }
 
+#[cw_serde]
+struct ValidatorReward {
+    validator_address: String,
+    reward: Vec<BankAmount>,
+}
+
+#[cw_serde]
+struct RewardsResponse {
+    rewards: Vec<ValidatorReward>,
+    total: Vec<BankAmount>
+}
+
 #[entry_point]
```

Comment:
Can use "DelegatorReward", it aligned with cosmos types. 
Tip: For cosmos-based msgs, you can research cosmwasm_std to see if the type is implemented or not, it saves some times drafting the structs